Journal of Extension.js

```diff
@@ -0,0 +1,360 @@
+/*
+Journal of Extension Translator
+Copyright (C) 2015 John Weis
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { 
+	return FW.doWeb(doc, url); }
+
+
+
+FW.Scraper({
+  itemType			:'journalArticle',
+  detect			:FW.Url().match(/\w+\d+(\.php|\.shtml)$/),
+  title				:FW.Xpath('//div[@class="DIV-title"]').text().trim(),
+  creators			:FW.Xpath('//div[@class="DIV-author"]/p/b|//div[@class="DIV-author"]/p/strong').text().cleanAuthor("author"),
+  publicationTitle	:"Journal of Extension",
+  abstractNote		:FW.Xpath('//div[@class="DIV-abstract"]/p').text().trim(),
+  date				:FW.Xpath('//p[@class="DIV-number"]|//p[@class="DIV-number screen_only"]').text().split(/\/\//),
```

Comment:
the split here might work, but it's ugly (since it technically creates an array and FW just happens to force the first element to a string). Why not use a similar match as you do for volume and issue below.